src/tests/multihost/ipa/test_subid_ranges.py

```diff
@@ -0,0 +1,125 @@
+                                                       f"subid-find"
+                                                       f"  --owner  "
+                                                       f"{user}")
+    user_details = result1.readlines()
+    global uid_start, uid_range, gid_start, gid_range
+    uid_start = int(user_details[5].split(': ')[1].split('\n')[0])
+    uid_range = int(user_details[6].split(': ')[1].split('\n')[0])
+    gid_start = int(user_details[7].split(': ')[1].split('\n')[0])
+    gid_range = int(user_details[8].split(': ')[1].split('\n')[0])
+    ssh1.close()
+
+
+@pytest.mark.usefixtures('environment_setup',
+                         'subid_generate',
+                         'bkp_cnfig_for_subid_files')
+@pytest.mark.tier2
+class TestSubid(object):
+    """
+    This is for ipa bugs automation
+    """
+    def test_subid_feature(self, multihost):
+        """
+        :Title: support subid ranges managed by FreeIPA
+        :id: 50bcdc28-00c8-11ec-bef4-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1803943
+        :steps:
+            1. Test newuidmap command
+            2. Test newgidmap command
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        ipa_subid_find(multihost)
+        ssh1 = SSHClient(multihost.client[0].ip,
+                         username=user, password=test_password)
+        (results1, results2, results3) = ssh1.exec_command("unshare"
+                                                           " -U bash"
+                                                           " -c 'echo $$"
+                                                           ">/tmp/unshare.pid;"
+                                                           "sleep 1000'")
+        time.sleep(2)
+        proces_id = int(execute_cmd(multihost,
+                                    "cat "
+                                    "/tmp/unshare.pid").stdout_text.strip())
+        uid = 0
+        gid = 1000
+        count = 1
+        (std_out, std_err, exit_status) = ssh1.exec_command(f"newuidmap "
+                                                            f"{proces_id}"
+                                                            f" {uid}"
+                                                            f" {uid_start}"
+                                                            f" {count}")
+        for i in exit_status.readlines():
+            assert "write to uid_map failed" not in i
+        (result, result1, exit_status) = ssh1.exec_command(f"newgidmap "
+                                                           f"{proces_id} "
+                                                           f"{gid} "
+                                                           f"{gid_start} "
+                                                           f"{count}")
+        for i in exit_status.readlines():
+            assert "write to uid_map failed" not in i
```

Comment:
This should look for `... gid_map ...` (not `uid_map`), right?

Reply:
Thats true , will fix it now